server/controllers/assessmentDayDataController/index.js

```diff
@@ -1,11 +1,12 @@
 import AssessmentDayDataModel from '../../models/AssessmentDayDataModel'
+import deepEqual from 'deep-equal'
 import ToCModel from '../../models/ToC'
-import AssessmentDayDataService from '../../services/AssessmentDayDataService'
 
 const AssessmentDayDataController = {
   create: async (req, res) => {
     try {
       const { dataDb } = req.app.locals
+      const updatedData = []
```

Comment:
this is a pretty generic variable name, what do you think about something more specific like `updatedParticipants`?

Reply:
[8be9854](https://github.com/AMP-SCZ/dpdash/pull/440/commits/8be9854ee40a699b58358200eaf2b168531f7582)